frontend/src/metabase/dashboard/components/Dashboard/stickyParameters.js

```diff
@@ -34,11 +36,16 @@ const checkIfShouldToggleStickiness = (dashboard, shouldBeSticky) => {
 };
 
 const checkIfParametersWidgetShouldBeSticky = dashboard => {
+  const isStickyForDevice = !(
+    dashboard.state.parametersList &&
+    dashboard.state.parametersList.length > MAXIMUM_PARAMETERS_FOR_STICKINESS &&
+    isSmallScreen()
+  );
```

Comment:
This makes it impossible to make filters sticky on desktop devices

Reply:
I don't think so... because the statement in the brackets is catering for when it is should not be sticky:
- IF it has `parametersList`
- AND `parametersList` is greater than 6
- AND it is a small screen
- THEN !(it is not sticky)

```
const isStickyForDevice = !(...);
```

Reply:
@losrebellos apologies, you're right, it works as expected! Just realized it was a different issue that made me think something's off with `isStickyForDevice`. For some reason, if I start editing a dashboard, add a few filters, save and then start scrolling, filter stickiness starts lagging (although it works fine if I open a dashboard with filters already / refresh the page). Please take a look:

**Steps to reproduce**

1. Open a dashboard with 4-5 filters
2. Click the "pencil" icon
3. Add 3-4 more filters (no need to connect them to card fields)
4. Save the changes
5. Start scrolling
6. Notice the filters don't become sticky for quite some time
7. Refresh the page and now it works as fast as it usually

### Demo

**This branch**

https://user-images.githubusercontent.com/17258145/165484234-f95f7c96-ddbd-4a43-8b98-f3056b6e12ef.mp4

**Master branch**

https://user-images.githubusercontent.com/17258145/165484245-4c18d51a-ef82-462b-9e2d-6161f716de5f.mp4